sys/newlib_syscalls_default/syscalls.c

```diff
@@ -516,3 +535,18 @@ int _gettimeofday_r(struct _reent *r, struct timeval *restrict tp, void *restric
     return -1;
 }
 #endif
+
+/**
+ * Current process times (not implemented).
+ *
+ * @param[out]  ptms    Not modified.
+ *
+ * @return  -1, this function always fails.
+ */
+clock_t _times(struct tms *ptms)
+{
+    (void)ptms;
+
+    errno = EINVAL;
```

Comment:
Shouldn't this also use the errno variable from struct reent?

Reply:
I copied it from libgloss, which is part of newlib.